packages/reports/src/Containers/open-positions.tsx

```diff
@@ -650,7 +650,8 @@ const OpenPositions = observer(({ component_icon, ...props }: TOpenPositions) =>
             />
         );
     };
-
+    // TODO: Uncomment and update this when DTrader 2.0 development starts:
+    // if (useFeatureFlags().is_dtrader_v2_enabled) return <Text size='l'>I am Open positions for DTrader 2.0.</Text>;
```

Comment:
Just a note: @maryia-deriv 
We should be handling the new redesign as a different component all together so that we can lazy load and bundle it separately.